prototyping/test_clibrary.py

```diff
@@ -12,18 +12,19 @@
 
 
 @pytest.fixture
-def wrapper():
+def cdll():
     source = SRC / "clibrary.cpp"
     compile_lib(source)
 
     yield CDLL(str(source.with_suffix(".so")))
 
 
-def test_get_point(wrapper, capsys):
-    wrapper.getPoint.restype = ctypes.POINTER(Point)
-    wrapper.getPoint.argtype = ctypes.POINTER(Point)
+def test_get_point(capsys):
+    cdll = lib
+    cdll.getPoint.restype = ctypes.POINTER(Point)
+    cdll.getPoint.argtype = ctypes.POINTER(Point)
     point_a = Point(10, 20)
-    p = wrapper.getPoint(point_a)
+    p = cdll.getPoint(point_a)
 
     # Print point members and assert values are right
     print(p.contents.x, p.contents.y)
```

Comment:
We get `(0, 0)` instead of `(20, 30)` for `(p.contents.x, p.contents.y)`. I am currently trying to understand why. I confirmed that the C++ function `getPoint` returns the right values from the C++ side, so the values are becoming zero in the conversion to Python, I think.

Reply:
@guadiromero Just to be sure: The exact same code built with gcc works, but built with qmake, it fails?